### PR TITLE
Update README.md Installation Guide

### PR DESCRIPTION
**Pull request by deepu105**
### 🔧 Changes
- Update README.md Installation Guide

### 📝 Checklist

- [ ] All new/changed/fixed functionality is covered by tests (or N/A)
- [ ] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Build, manage and test your [Auth0](https://auth0.com/) integrations from the co
 
 ## Installation
 
-### macOS
+### macOS & Linux
```
Comment (willvedd) [inline README.md:35]: There is a linux section below. Maybe this heading should just be "brew".

Comment (deepu105) [inline README.md:35]:
```suggestion
### Homebrew (macOS & Linux)
```

**Comment by deepu105** [comment]
@developerkunal can we merge this

**Review by developerkunal — APPROVED** [review]
LGTM

**Pull request merged**